contracts/services/PayoutManager.sol

```diff
@@ -0,0 +1,214 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./IMerkleTreePathValidator.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+
+contract PayoutManager {
+
+    using SafeMath for uint256;
```

Comment:
You don't need `SafeMath` after solidity version 0.8.0.

Reply:
Good to know 👍